index.bs

```diff
@@ -690,7 +694,7 @@ authorizing an authenticator with which to complete the operation.
     };
 </pre>
 
-<div dfn-for="Attachment">
+<div>
```

Comment:
No good reason; I just missed that I should remove it once it didn't have a `dfn-for`. I've pushed a new commit that removes it. Thanks!